packages/react-components/src/types/Attachment.ts

```diff
@@ -33,6 +33,15 @@ export interface AttachmentMetadata {
   payload?: Record<string, string>;
 }
 
+/**
+ * @beta
```

Comment:
Should `AttachmentMenuAction` be marked as internal or private instead as it's not a part of the public API?